includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -120,4 +138,50 @@ public function capture_terminal_payment( $request ) {
 			return new WP_Error( 'wcpay_server_error', __( 'Unexpected server error', 'woocommerce-payments' ), [ 'status' => 500 ] );
 		}
 	}
+
+	/**
+	 * Returns customer id from order. Create or update customer if needed.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
```

Comment:
Nit: Can we have `return` and `throw` in the comment here?

Reply:
Good catch. Added `@return` but not `@throws` as there is try-catch block, so this function won't throw any exception.